tests/src/proc_macros.rs

```diff
@@ -1,3 +1,6 @@
+// Not all proc macros are used let's suppress it for now.
+#![allow(dead_code)]
```

Comment:
Isn't there an `unused_macros` lint we can use instead?

Reply:
I think it only works for `declarative macors (macro_rules!)`, didn't work in this case